index.html

```diff
@@ -138,7 +139,9 @@ <h6>1,000 views | 1 year ago</h6>
             </div>
           
         </div>
+        </section>
 
+        <section>
         <div id="video-container">
             <div class="video-template">
                 <img class="video-image" src="https://picsum.photos/400/200" alt="">
```

Comment:
## Please add to all images the alt reference.

Please add the alt tag for better SEO and good html practices.